dns-zone.tf

```diff
@@ -0,0 +1,12 @@
+data "azurerm_dns_zone" "play" {
+  name                = "us-hpccsystems-dev.azure.lnrsg.io"
```

Comment:
This is a hardcoded name.  Should it be added as a Terraform variable instead?  That is likely, since you may have to supply a resource group for the DNS Zone as well.

Reply:
> This is a hardcoded name. Should it be added as a Terraform variable instead? That is likely, since you may have to supply a resource group for the DNS Zone as well.

The 'data' block has been replaced with a "module" block and nothing in the module block is hardcoded.